### ISSUE TITLE
libraries

### ISSUE THREAD

**Issue by Kuhicop** [m-rogalski90/SimpleTibiaBot#1]
Hello it's not finding .h items.
Example:
could not open the source file (  error code "QtCore\qobject.h").

**Comment by m-rogalski90** [comment]
This project uses Qt framework as it's base. You can download it here: https://www.qt.io/download-qt-for-application-development

The Qt version needed for this is Qt 5.4.

**Comment by Kuhicop** [comment]
I can't find the project in Qt
https://gyazo.com/c5c656cbe8bf58ef08edc9646c6a1499

**Comment by m-rogalski90** [comment]
Using QtCreator you need a .pro ( a project description ) file. This one was made inside VisualStudio so it does not contain any .pro files. Open this project up in VisualStudio ( if applicable ) and change lib, include and linking paths to where your Qt is installed.

**Comment by Kuhitacop** [comment]
I installed Qt, which project type should I select?
I don't know which steps I should follow in Visual Studio, so maybe it's easier in Qt?

**Comment by Kuhicop** [comment]
I'm trying with the following part of the code: https://pastebin.com/QN2EE39m
I get the following error:

> E0167	an argument of type "const char *" is not compatible with a parameter of type "TCHAR *"
This is for the line 21 in pastebin DWORD baseAddress = getBaseAddress(pid, "Tibia.exe");

**Comment by mquinaz** [comment]
The error is just for setting your libraries in a wrong way in visual studio. I wanted to ask if the project works assuming you change to the proper adresses of the game in its current state / or another private server, thank you

**Comment by m-rogalski90** [comment]
Yes, If you use exact same version of the client it will work as it should.
